[domaincontrol.com] HTTPSConnectionPool(host='domaincontrol.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ffb35c867e0>: Failed to establish a new connection: [Errno 111] Connection refused'))
[macromedia.com] HTTPSConnectionPool(host='macromedia.com', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7ffb354f5bb0>, 'Connection to macromedia.com timed out. (connect timeout=None)'))
[mangosip.ru] HTTPSConnectionPool(host='mangosip.ru', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7ffb3587c230>, 'Connection to mangosip.ru timed out. (connect timeout=None)'))
[msedge.net] HTTPSConnectionPool(host='msedge.net', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7ffb35bb63c0>, 'Connection to msedge.net timed out. (connect timeout=None)'))
[nih.gov] ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
[shifen.com] HTTPSConnectionPool(host='shifen.com', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7ffb35a633b0>, 'Connection to shifen.com timed out. (connect timeout=None)'))
[telecid.ru] HTTPSConnectionPool(host='telecid.ru', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, "[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'telecid.ru'. (_ssl.c:1000)")))
